tests/rptest/tests/log_level_test.py

```diff
@@ -23,6 +23,22 @@ def __init__(self, *args, **kwargs):
         # it will start.
         super().__init__(*args, log_level=self.initial_log_level, **kwargs)
 
+    @cluster(num_nodes=3, log_allow_list=["Set log level for {assert}"])
+    def test_get_loggers(self):
+        admin = Admin(self.redpanda)
+        node = self.redpanda.nodes[0]
+        loggers = admin.get_loggers(node)
+        assert len(loggers) >= 42, f"Only {len(loggers)} loggers: {loggers}"
```

Comment:
instead of assuming the list will grow how about we test that the list contains some loggers that we believe with high probability are unlikely to go away (e.g. "seastar", "http", "kafka").

Reply:
Done